src/NativeScript/Metadata/Metadata.mm

```diff
@@ -112,6 +121,48 @@ static int compareIdentifiers(const char* nullTerminated, const char* notNullTer
     return nullptr;
 }
 
+const std::vector<const MethodMeta*> BaseClassMeta::members(const char* identifier, size_t length, MemberType type, bool includeProtocols, bool onlyIfAvailable) const {
+    ASSERT(type == MemberType::InstanceMethod || type == MemberType::StaticMethod);
+    const ArrayOfPtrTo<MemberMeta>* members;
+    switch (type) {
+    case MemberType::InstanceMethod:
+        members = &this->instanceMethods->castTo<PtrTo<MemberMeta>>();
+        break;
+    case MemberType::StaticMethod:
+        members = &this->staticMethods->castTo<PtrTo<MemberMeta>>();
+        break;
+    default:
+        members = nullptr;
+        ASSERT_NOT_REACHED();
+    }
+
+    std::vector<const MethodMeta*> result;
+    if (members != nullptr) {
+
+        for (int i = 0; i < members->count; i++) {
+
+            const MethodMeta* m = reinterpret_cast<const MethodMeta*>((*members)[i].valuePtr());
+            if ((strncmp(identifier, m->jsName(), length) == 0) && strlen(m->jsName()) == length) {
+                result.push_back(m);
+            }
+        }
+    }
+
```

Comment:
I think we may need to also scan the members of base classes. But I'm not sure if they're present in the current class' members or not. You should write a unit test and make it pass if it doesn't.

Reply:
Inherited class members are handled via the prototype chain and need not be considered here. Resolving comment.